apriltag/src/main/native/include/frc/apriltag/AprilTagFieldLayout.h

```diff
@@ -60,14 +64,25 @@ class WPILIB_DLLEXPORT AprilTagFieldLayout {
                       units::meter_t fieldLength, units::meter_t fieldWidth);
 
   /**
-   * Set the alliance that your team is on.
+   * Sets the origin based on a pre-known enumeration of positions. The position
+   * is calculated from values in the configuration file.
+   *
+   * This changes the GetTagPose(int) method to return the correct pose for your
+   * alliance.
+   *
+   * @param alliance The alliance to mirror poses for.
```

Comment:
Is it a mirror or a 180 degree rotation?

Reply:
The latter.

Reply:
We should probably say that then.

Reply:
Specifically, it's a 180° rotation around the center of the field.

Reply:
Let's PR this fix separately.